instawow/manager.py

```diff
@@ -145,11 +145,13 @@ def __missing__(self, key: Hashable) -> NoReturn:
 class Manager:
 
     RESOLVERS = {CurseResolver, WowiResolver, TukuiResolver, InstawowResolver}
+    SEARCH_RESOLVERS = {CurseResolver}
```

Comment:
I think a separate `SEARCH_RESOLVERS` set is redundant if we're catching `NotImplementedError`s below.

Reply:
This is what I was doing at first, but then I realized RESOLVERS contains `instawow` for some reason, so it resulted in infinite recursive calls. I'm not sure if that's a bug or if I'm doing something wrong.

Reply:
I don't see why `InstawowResolver` (that's the resolver used for instawow's own WeakAuras Companion add-on) might cause recursion errors.  Are you sure it wasn't something to do with not having a `search` function on the parent class?